src/sonolus/playlist/item.ts

```diff
@@ -19,6 +19,10 @@ export const updatePlaylistItems = () => {
             for (const characterIndex of level.meta.characterIndexes) {
                 playlist.meta.characterIndexes.add(characterIndex)
             }
+
+            if (level.meta.publishedAt > playlist.meta.publishedAt) {
```

Comment:
In cases where a playlist contains a mix of released and unreleased levels, it will not be visible, which players might still prefer to be able to see released levels.

Reply:
I trusted Copilot too much (sorry!), I reversed the sign.